src/analysis/import/import_jobs.sh

```diff
@@ -34,19 +34,53 @@ NB_POSTGRESQL_PASSWORD - Default: gis
 "
 }
 
-function import_job_data() {
+function fetch_census_data() {
+    PFB_JOB_URL="http://lehd.ces.census.gov/data/lodes/LODES7/${PFB_STATE}/od/${NB_JOB_FILENAME}.gz"
+    wget -nv -O "${JOB_DOWNLOAD}" "${PFB_JOB_URL}" 
+    WGET_STATUS=$?
+    set -e
+    # Recursively try prior years
+    if [[ $WGET_STATUS -eq 8 ]] && [[ $CENSUS_YEAR -gt 2017 ]]; then
+        (($CENSUS_YEAR--))
+        echo "No ${CENSUS_YEAR} job data available, falling back to ${CENSUS_YEAR-1} data..."
+        NB_JOB_FILENAME="${PFB_STATE}_od_${NB_DATA_TYPE}_JT00_${CENSUS_YEAR}.csv"
+        S3_PATH="s3://${AWS_STORAGE_BUCKET_NAME}/data/${NB_JOB_FILENAME}.gz"
+        JOB_DOWNLOAD="${NB_TEMPDIR}/${NB_JOB_FILENAME}.gz"
+        fetch_census_data
+        
+        if [ "${AWS_STORAGE_BUCKET_NAME}" ] && aws s3 ls "${S3_PATH}"; then
+            aws s3 cp "${S3_PATH}" "${JOB_DOWNLOAD}"
+            echo "Downloaded job data file from S3"
+        else
+            wget -nv -O "${JOB_DOWNLOAD}" "${PFB_JOB_URL}"
+            if [ "${AWS_STORAGE_BUCKET_NAME}" ]; then
+                echo "Uploading job data file to S3 cache"
+                aws s3 cp "${JOB_DOWNLOAD}" "${S3_PATH}"
+            fi
+        fi
+    elif [ "${AWS_STORAGE_BUCKET_NAME}" ]; then
+        echo "Uploading job data file to S3 cache"
+        aws s3 cp "${JOB_DOWNLOAD}" "${S3_PATH}"
+    fi
+}
 
+function import_job_data() {
+    NB_DATA_TYPE="${3:-main}"    # Either 'main' or 'aux'
     ROOT_TEMPDIR="${NB_TEMPDIR:-$(mktemp -d)}"
     NB_TEMPDIR="${ROOT_TEMPDIR}/import_jobs"
     mkdir -p "${NB_TEMPDIR}"
     # Dir and files must be world readable/executable for postgres to use copy command
     # Must chmod after creating subdir
     chmod -R 775 "${ROOT_TEMPDIR}"
 
-    NB_STATE_ABBREV="${1}"
-    NB_DATA_TYPE="${2:-main}"    # Either 'main' or 'aux'
-
-    NB_JOB_FILENAME="${NB_STATE_ABBREV}_od_${NB_DATA_TYPE}_JT00_2018.csv"
+    PFB_COUNTRY="${1}"
+    PFB_STATE="${2}"
+    if [ -n "${PFB_JOB_URL}" ]; then
+        NB_JOB_FILENAME=$(basename "${PFB_JOB_URL}")
```

Comment:
If `PFB_JOB_URL` is already set, we should be downloading the file from that URL.  This is using the URL to get the filename, then looking for the filename in the `/data/` directory of the configured storage bucket.  But we're asking for a whole URL, not just a filename, so it doesn't make sense to reconfigure it like that.  At best it will happen to match, but generally (like if Rebecca is using a file in a bucket she made, or if I'm using a file in your bucket) the file won't be found at the new S3 path.

Also, currently `basename "https://jacob-pfb-storage-us-east-1.s3.amazonaws.com/data/be_od_main_JT00_2017.csv.gz"` returns `be_od_main_JT00_2017.csv.gz`, then we add another `.gz` below when we set `S3_PATH`.  It looks like we could add an argument to get this to also strip the `.gz` extension if it's there:
```suggestion
        NB_JOB_FILENAME=$(basename "${PFB_JOB_URL}" .gz)
```

I guess we do need to parse out the filename, since we need to point the psql `\copy` command at it further down, but I think it probably doesn't make sense to treat it as meaningful the way we do for the US files.